tools/reactive-controllers/src/RovingTabindex.ts

```diff
@@ -45,18 +45,17 @@ export class RovingTabindexController<
     }
 
     manageTabindexes(): void {
-        if (this.focused) {
+        if (this.focused && !this.delegatesFocus) {
```

Comment:
Ideally we want to look at the components that use these controllers to see if they are following WAI ARIA APG for keyboard nav, but in the meantime, so we don't have scope-creep on this PR, I'm allowing existing components to still follow this pattern.